reactor-netty-core/src/main/java/reactor/netty/transport/ClientTransportConfig.java

```diff
@@ -177,14 +174,15 @@ protected ClientTransportConfig(ClientTransportConfig<CONF> parent) {
 		this.proxyProvider = parent.proxyProvider;
 		this.remoteAddress = parent.remoteAddress;
 		this.resolver = parent.resolver;
-		this.defaultResolver = parent.defaultResolver;
 	}
 
 	@Override
 	protected Class<? extends Channel> channelType(boolean isDomainSocket) {
 		return isDomainSocket ? DomainSocketChannel.class : SocketChannel.class;
 	}
 
+	protected abstract AddressResolverGroup<?> defaultAddressResolverGroup();
```

Comment:
would probably benefit from some javadoc, and more importantly also document the concrete cases (eg. `HttpConfig` would mention that the default comes from `HttpResources` and is thus shared between all such configs)

Reply:
done